ast/TreeSanityChecks.cc

```diff
@@ -82,7 +82,6 @@ void If::_sanityCheck() {
 }
 
 void InsSeq::_sanityCheck() {
-    ENFORCE(!stats.empty());
```

Comment:
Why are we removing these ENFORCEs?

Reply:
because now there will be empty `InsSeq`s

Reply:
https://github.com/sorbet/sorbet/pull/1967#discussion_r335242292

^ Could we keep this ENFORCE if we made a change here?